src/Components/MainHeader/MiddleMenu/MiddleMenu.js

```diff
@@ -37,12 +27,29 @@ function MiddleMenu() {
             <form className="Middle-inner-area-attr">
               <fieldset className="Middle-inner-contents-area">
                 <div className="Middle-menu-list">
-                  {contents.map((content) => (
-                    <MenuList content={content} />
-                  ))}
+                  <label>
+                    <input
+                      className="Menu-btn-atr"
+                      type="radio"
+                      onClick={LodgeView}
+                    ></input>
+                    <span className="Name-wrapper">숙소</span>
+                  </label>
+                  <label>
+                    <input
+                      className="Menu-btn-atr"
+                      type="radio"
+                      onClick={ExperienceView}
+                    ></input>
+                    <span className="Name-wrapper">체험</span>
+                  </label>
+                  <label>
+                    <input className="Menu-btn-atr" type="radio"></input>
+                    <span className="Name-wrapper">온라인 체험</span>
+                  </label>
                 </div>
               </fieldset>
-              <Lodge />
+              <div>{state}</div>
```

Comment:
위에서 이부분을 상황에 따라 다른 컴포넌트를 하고 싶었군요
state를 boolean으로 설정하고 삼항연산자를 사용해서 상태에 따른 컴포넌트를 나타낼 수 있습니다.
예를들어 showLodge란 state가 true일때 `<Lodge />`, false일 떄 `<Experience />`를 나타내고 싶다면
```suggestion
              {` showLodge ? <Lodge /> : <Experience />}
```
로 나타낼 수 있습니다